python/tidy/servo_tidy/tidy.py

```diff
@@ -723,13 +725,32 @@ def parse_config(content):
     config['ignore']['directories'] = map(lambda path: os.path.join(*path.split('/')),
                                           config['ignore']['directories'])
 
+    # Add dict of dir, list of expected ext to config
+    dirs_for_ext = config_file.get("check_dirs_for_exts", {})
```

Comment:
By the way, the `get` with default `{}` is when accessing the parsed `toml` file (see(this line)[https://github.com/servo/servo/blob/master/python/tidy/servo_tidy/tidy.py#L712]), 
as opposed to the python `config` object that is (declared here)[https://github.com/servo/servo/blob/master/python/tidy/servo_tidy/tidy.py#L27]

So if I remove the `check_ext` key from the default config(the python one), then at some point in `parse_config` I will have to do a `config['check_ext'] = {}`. 

Perhaps actually 'declaring' this key at the top of the page, is more explicit than doing so somewhere in code on the page?